website/docs/src/content/cloud/observability.mdx

```diff
@@ -0,0 +1,269 @@
+- **Status indicators**: Quickly spot errors or warnings with color coding
+- **Component filtering**: Focus on specific components or component types
+
+Click on any component in the flame graph to inspect its details, including inputs, outputs, and timing information.
+
+### Viewing component inputs and outputs
+
+For each component in your workflow, you can inspect:
+
+1. **Input properties**: All props passed to the component
+2. **Output values**: The data returned by the component
+3. **Execution duration**: How long the component took to execute
+4. **Metadata**: Additional information like token counts for LLM calls
+
+![Component trace](/cloud/component-trace.png)
+
+This visualization is particularly valuable for debugging production and user-reported issues like hallucinations.
+
+### Viewing historical traces
+
+The GenSX Console maintains a history of all your workflow executions, allowing you to:
+
+- **Compare executions**: See how behavior changes across different runs
+- **Identify patterns**: Spot recurring issues or performance bottlenecks
+- **Filter by status**: Focus on successful, failed, or in-progress executions
+- **Search**: Find historical executions
+
+Historical traces are automatically organized by project and environment, making it easy to find relevant executions.
+
+## Configuring traces
+
+tGenSX provides flexible options for configuring and organizing traces for the GenSX Cloud serverless platform, local development, and any other deployment platform like vercel, cloudflare and AWS.
+
+### Tracing GenSX Cloud workflows
+
+When running workflows on GenSX Cloud, tracing is automatically configured:
+
+- **Project context**: Traces are associated with the correct project
+- **Environment segregation**: Development, staging, and production traces are kept separate
+- **Authentication**: API keys and organization information are handled automatically
+- **Retention**: Traces are stored according to your plan limits
+
+No additional configuration is needed – everything works out of the box.
+
+### Tracing on other deployment platforms
+
+To enable tracing for workflows deployed outside of GenSX Cloud (like AWS Lambda, Google Cloud Functions, etc.), you need to set several environment variables:
+
+```bash
+# Required variables
+GENSX_API_KEY=your_api_key_here
+GENSX_ORG=your_gensx_org_name
+GENSX_PROJECT=your_project_name
+
+# Optional variables
+GENSX_ENVIRONMENT=your_environment_name        # Separate traces into specific environments
+GENSX_CHECKPOINTS=false                        # Explicitly disable
+```
+
+### Configuring traces for local development
+
+For local development, tracing configuration is automatically inferred from:
+
+1. The `gensx.yaml` file in your project root
+2. Your local configuration managed by the `gensx` CLI in `~/.config/gensx/config`
+3. Optionally the `GENSX_ENVIRONMENT` environment variable to separate local traces from other environments
```

Comment:
```suggestion
3. Optionally the `GENSX_ENVIRONMENT` environment variable can be set to separate local traces from other environments
```

Does this actually work though? do we create the environment for you when you do this?

Reply:
Outstanding TODO we need to fix before launch: https://github.com/gensx-inc/gensx/issues/583